paddle/fluid/operators/strided_memcpy.h

```diff
@@ -58,6 +58,7 @@ inline void StridedNumelCopyWithAxis(const platform::DeviceContext& ctx,
   int64_t before = dst_stride_numel[0] / dst_stride_numel[axis];
   int64_t src_after = src_stride_numel[axis];
   int64_t dst_after = dst_stride_numel[axis];
+  int64_t copy_size = std::min(src_after, dst_after);
```

Comment:
As the offline discuss, `min` is confusion, maybe pass the `copy_size` as a parameter is a better way.